configure

```diff
@@ -32895,6 +32926,11 @@ if test "$wxUSE_PROFILE" = "yes" ; then
     PROFILE_FLAGS=" -pg"
 fi
 
+PIC_FLAGS=
+if test "$wxUSE_PIC" = "yes" ; then
+    PIC_FLAGS=" -fpic"
+fi
+
```

Comment:
We already have `PIC_FLAG` defined in `AC_BAKEFILE_SHARED_LD` from `build/aclocal/bakefile.m4` (which is, wrongly but conveniently for us, called unconditionally, i.e. even when building static libraries), so I think we should use it instead (even if `-fpic` is potentially more efficient). Is there any reason you have added this one or was it just an oversight?

Reply:
I didn't know I could use that one. But that file is only included after most of the compiler arguments are set, so I can't add it to `C_AND_CXX_FLAGS` like I do now, but have to do something like
```
if test "$wxUSE_PIC" = "yes" ; then
    CFLAGS=`echo $CFLAGS $PIC_FLAG `
    CXXFLAGS=`echo $CXXFLAGS $PIC_FLAG `
    OBJCFLAGS=`echo $OBJCFLAGS $PIC_FLAG `
    OBJCXXFLAGS=`echo $OBJCXXFLAGS $PIC_FLAG `
fi
```

Should I also add it to `WXCONFIG_CPPFLAGS`?

Reply:
No, I don't think this should go into `WXCONFIG_CPPFLAGS` (and it's not a cpp flag anyhow), there doesn't seem to be any good reason to force compiling any code using wx as PIC, if the application wants to use PIC (or, more likely, PIE) flag for its code, it should do it on its own.